### PR TITLE
ADD ES language

### PR DESCRIPTION
**Pull request by andresin87**
Spanish first approach language version.

Work in progress [add npm flag icons and modify the language picker component to use this flags with css (not as an image) in order to easy add new languages].

Then I'll include catalan.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "express": "4.13.4",
     "extract-text-webpack-plugin": "1.0.1",
     "file-loader": "0.8.5",
+    "flag-icon-css": "^2.3.0",
```
Comment (antoinejaussoin) [inline package.json]: This is an excellent idea

**Comment by antoinejaussoin** [comment]
I don't know how much work it would be for you, but do you think you could target the "develop" branch for this PR? I'm trying to keep master for released code only.

About the flag-icon-css, this is an excellent idea, will have a look!!

**Comment by andresin87** [comment]
Ok. I'll do this weekend.

**Comment by antoinejaussoin** [comment]
Brilliant thanks!!

**Comment by antoinejaussoin** [comment]
Hi, I implemented your idea for flag-icon-css, and merged to develop. Any luck with your ES translations in develop?

**Pull request closed without merging**